sqlrunner/main.py

```diff
@@ -44,8 +44,12 @@ def main(args):
 def generate_column_headers(dummy_data_file, sql_query):
     tsql_parser = TSQL()
     parsed = [q for q in tsql_parser.parse(sql_query) if q]
-    # expand out "*" to full column names
+    # expand out "*" to full column names where possible
     columns = qualify_columns(parsed[0], schema=None, infer_schema=True).named_selects
+    if "*" in columns:
+        raise SyntaxError(
```

Comment:
`SyntaxError` doesn't feel right. The syntax may be correct, it's just that SQL Runner can't handle it correctly. Elsewhere, we raise `RuntimeError`s for "Sorry, no can do" situations.

Reply:
Yeah there didn't seem to be an exact fit, but I agree Runtime is closer!